enhancements/update/change-management-and-maintenance-schedules.md

```diff
@@ -0,0 +1,1220 @@
+| `true`         | *      | *       | Paused indefinitely |
+| `null`/`false` | set    | `null`  | Permissive during reoccurring windows time. Paused at all other times.                                                                                                                  |
+| `null`/`false` | set    | set     | Permissive during reoccurring windows time modulo excluded date ranges during which it is paused. Paused at all other times.                                                                                                                                                                                                         |
+| `null`/`false` | `null` | set     | Permissive except during excluded dates during which it is paused.                                                                                                                             |
+| date           | *      | *       | Honor permit and exclude values, but only after the specified date. For example, permit: `null` and exclude: `null` implies the strategy is indefinitely permissive after the specified date. |
+
+
+#### MachineConfigPool Assisted Strategy Configuration
+
+```yaml
+spec:
+  changeManagement:
+    strategy: Assisted
+    config:
+      assisted:
+        permit:
+          # The assisted strategy will allow the MCO to process any rendered configuration
+          # that was created before the specified datetime. 
+          renderedConfigsBefore: <datetime>
+          # When AllowSettings, rendered configurations after the preceding before date
+          # can be applied if and only if they do not contain changes to osImageURL.
+          policy: "AllowSettings|AllowNone"
+```
+
+The primary user of this strategy is `oc` with tentatively planned enhancements to include verbs 
+like:
+```sh
+$ oc adm update worker-nodes start ...
+$ oc adm update worker-nodes pause ...
+$ oc adm update worker-nodes rollback ...
+```
+
+These verbs can leverage the assisted strategy and `pausedUntil` to allow the manual initiation of worker-nodes
+updates after a control-plane update. 
+
+#### MachineConfigPool Manual Strategy Configuration
+
+```yaml
+spec:
+  changeManagement:
+    strategy: Manual
+    config:
+      manual:
+        desiredConfig: <rendered-configuration-name>
+```
+
+The manual strategy requests no automated initiation of updates. New and rebooting
+nodes will only receive the desired configuration. From a metrics perspective, this strategy
+is always paused state.
+
+#### Metrics
+
+`cm_change_pending`
+Labels:
+- kind=ClusterVersion|MachineConfigPool|HostedCluster|NodePool
+- object=<object-name>
+- system=<control-plane|worker-nodes>
+
+Value: 
+- `0`: no material changes are pending.
+- `1`: changes are pending but being initiated.
+- `2`: changes are pending and blocked based on this resource's change management policy.
+- `3`: changes are pending and blocked based on another resource in the change management hierarchy.

```

Comment:
It would require docs. 0/1 values to indicate bools are common. I went beyond in this EP because I feel this area is ripe for "compliance" alerts; i.e. the administrator wants to know that their maintenance schedule is configured in the way they expect in order to prevent unexpected workload disruption and or want their operations teams to be aware of any temporary divergence (a fixed admission policy may be too rigid).
The metrics, therefore, try to communicate the operative aspects of the overall configuration, instead of simple status, to help an administrator craft alerts when expectations are violated. 
This thought process applied most to `cm_strategy_enabled`. I then used the same concept to provide some level of diagnostic insight for pending changes in `cm_change_pending`, but I'm less passionate about preserving the ability to alert on these particular values.